src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -10,10 +10,20 @@ namespace Microsoft.Extensions.Logging.Console
 {
     public class ConsoleLogger : ILogger
     {
+        // Multiple logger instances can be created during an application's lifetime
```

Comment:
I'd say something different here - this doesn't really capture why this is a static. Mention:
- write aren't atomic 
- multiple loggers will be created with different names
- console is global so a static is a safe assumption